denylist/src/denylist.rs

```diff
@@ -51,6 +56,24 @@ impl DenyList {
         })
     }
 
+    pub fn from_filter<P: AsRef<Path>>(filter_path: P) -> Result<Self> {
```

Comment:
given this code is shared pretty much as is with the `new()` function, be good to spin it out into a shared function and used by both new() and from_filter()

Reply:
Fair enough, I'll update shortly

Reply:
Updated [a1ba4fd](https://github.com/helium/oracles/pull/362/commits/a1ba4fdba49ee939d75ac10fae9ee147bf8eb6ba)